middle_end/flambda/terms/code_size.rec.ml

```diff
@@ -434,9 +434,14 @@ let increase_due_to_let_cont_recursive ~size_of_handlers =
 let rec expr_size ~find_code expr size =
   match Expr.descr expr with
   | Let let_expr ->
-    let size = named_size ~find_code (Let_expr.defining_expr let_expr) size in
     Let_expr.pattern_match let_expr
-      ~f:(fun _bindable_let_bound ~body -> expr_size ~find_code body size)
+      ~f:(fun bindable_let_bound ~body ->
+        let kind = Bindable_let_bound.name_mode bindable_let_bound in
```

Comment:
`kind` is a wrong name here, everywhere else in the code `kind` refers to a `Flambda_kind.t`, which makes these lines a bit confusing to read, as in other places, `name_mode` would be a batter name, ^^

Reply:
I've also renamed `kind` to `name_mode` in `terms/let_expr.rec.ml` ( I copy-pasted the code snippet from it...).